Eatery/Controllers/Eateries/ARViewController.swift

```diff
@@ -105,7 +105,11 @@ class ARViewController: UIViewController, CLLocationManagerDelegate, SceneLocati
     func displayEateries() {
         guard let location = sceneLocationView.currentLocation() else { return }
 
-        for eatery in eateries.filter({ $0.location.distance(from: location) / metersInMile < 0.5 }) {
+        let closeEateries = eateries.filter { eatery -> Bool in
+            return location.distance(from: eatery.location, in: .miles) < 0.5
```

Comment:
can you simplify this to `eateries.filter { location.distance(from: $0.location, in: .miles) < 0.5 }`